src/modules/infobox/components/FeatureElevationProfile.vue

```diff
@@ -80,7 +80,7 @@ function onCSVDownload() {
         <div v-if="!hasData">
             <LoadingBar v-if="!profileRequestError" />
             <span v-else class="text-danger" data-cy="profile-error-message">
-                {{ i18n.t(profileRequestError.messageKey) }}
+                {{ i18n.t(profileRequestError.message) }}
```

Comment:
This will probably break other error message like PofileError  just above the throw of outofbounderror. Also the error should be translated in all languages.